### PR TITLE
uutils-diffutils 0.4.1 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by pthariensflame** [comment]
Ah, right, notability.  Would it be better to admit this as part of `uutils` (otherwise sufficiently notable, with `uutils-coreutils` and `uutils-findutils` already accepted), or would it be better to wait?

**Review by ZhongRuoyu — APPROVED** [review]
Given its close relationship to other uutils projects, I'm okay to accept it as-is.

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
yeah, that is more like a temperature check, agree that we can merge as it is.

**Comment by chenrui333** [comment]
also need to rename the binaries with prefix `u` like the other coreutils formulae.

File: Formula/u/uutils-diffutils.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  head "https://github.com/uutils/diffutils.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(root: libexec)
+    mv libexec/"bin", libexec/"uubin"
+    Dir.children(libexec/"uubin").each do |cmd|
+      bin.install_symlink libexec/"uubin"/cmd => "u#{cmd}"
+    end
```
Comment (chenrui333) [inline Formula/u/uutils-diffutils.rb:16]: you might also need a caveat update for this too.

Comment (pthariensflame) [reply Formula/u/uutils-diffutils.rb:16]: What should it say?

Comment (chenrui333) [reply Formula/u/uutils-diffutils.rb:16]: yeah, model it after the other two. 👍

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@pthariensflame thanks a lot for your efforts!

**Pull request merged**